env.dist

```diff
@@ -3,10 +3,7 @@ AWS_QUEUE_REGION=
 DEBUG=false
 INCOMING_QUEUE_URL=
 OUTGOING_QUEUE_URL=
-WORKER_ARCHIVER_CONNECTION_STRING=postgresql://localhost:5432/sawmill
 MOFO_STAFF_EMAIL=
-SFR_SPREADSHEET=1swJ87dr03nxhYvn7uJoWpybE3kcLV2bW3OGoJ-H-z3o
-SFR_WORKSHEET=0
 TEACH_CLIENT_ID=
 
 # web config
```

Comment:
A few more vars we could add: STRIPE_SECRET, COINBASE_SECRET, STRIPE_SECRET_KEY

Reply:
we don't have a coinbase secret for testing, so I'll just add in the testing keys for Stripe.